lib/news_scraper/transformers/article.rb

```diff
@@ -40,14 +42,25 @@ def transformed_response
       def parsed_data(data_type)
         return nil unless scrape_details[data_type]
 
-        scrape_pattern = scrape_details[data_type]['pattern']
         scrape_method = scrape_details[data_type]['method'].to_sym
-
-        noko_html = Nokogiri::HTML(payload)
-
-        Sanitize.fragment(
-          noko_html.send(scrape_method, scrape_pattern)
-        ).squish
+        case scrape_method
+        when :xpath, :css
+          scrape_pattern = scrape_details[data_type]['pattern']
+          noko_html = Nokogiri::HTML(payload)
+          Sanitize.fragment(
+            noko_html.send(scrape_method, scrape_pattern)
+          ).squish
+        when :readability
+          content = Readability::Document.new(
+            @payload,
+            remove_empty_nodes: true,
+            tags: %w(div p img a table tr th tbody td h1 h2 h3 h4 h5 h6),
```

Comment:
we can probably remove `img`, but it might be interesting later on. The rest carry context such as headings, table data, data... should we add `span` or...